app/uk/gov/hmrc/agentsubscriptionfrontend/models/AMLSDetails.scala

```diff
@@ -16,10 +16,11 @@
 
 package uk.gov.hmrc.agentsubscriptionfrontend.models
 
-import org.joda.time.LocalDate
+import java.time.LocalDate
+
 import play.api.libs.json.Json
 
-case class AMLSDetails(amlsBodyName: String, membershipNumber: String, expiryDate: LocalDate)
+case class AMLSDetails(supervisoryBody: String, membershipNumber: String, membershipExpiresOn: LocalDate)
```

Comment:
it makes sense to have only these three fields in the models - so adjusted the model in the backend `agent-subscription` and `agent-assurance` too ,  
look at `case class CreateAmlsRequest(utr: Utr, amlsDetails: AmlsDetails)` in the agent assurance